frontend/client.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_API_BASE_URL=http://localhost:5000
+REACT_APP_CLIENT_BASE_URL=http://localhost:80
```

Comment:
Remove these keys. You need to merge with the latest dev. We've got rid off these keys compeltely.